apps/tailwind-components/components/input/Search.vue

```diff
@@ -22,21 +28,36 @@ function handleInput(input: string) {
     emit("update:modelValue", input);
   }, 500);
 }
+
+const HEIGHT_MAPPING = {
```

Comment:
We may want to adjust the size of the icons as well. The width/height does not seem proportional to the size of the input element.

Reply:
done

Reply:
That looks a lot more balanced.